docs/self-managed/installation-methods/helm/upgrade/helm-870-880.md

```diff
@@ -185,7 +185,38 @@ The following entities will be migrated:
 
 Once the migration is complete, you can safely delete the job, its associated ConfigMap, and the migration client in Management Identity.
 
-### 5. Run the Upgrade
+### 5. Process Migration
+
+The Process Migration app migrates from the old, separated indices in Elasticsearch or OpenSearch on 8.7 to the new unified indices on 8.8.
+
+In 8.7 and previous versions there were two indices:
+
+- `tasklist-process`
+- `operate-process`
+
+From 8.8 onwards, there will only be one `operate-process` index for the needs of the entire Camunda Cluster.
+
+:::warning
+
+- The process migration can't be ran on a fresh install of 8.8. The migration will only be successful on upgrade from 8.7.
+- The migration can only run once.
+- Before upgrading, the 8.7 zeebe workers should already have some load and should NOT be idle. Otherwise the migration will fail.
```

Comment:
Can we add link here to detect and deal with this faluire?

Reply:
What do you mean by a link? I don't think we have another doc like this. I will resolve this for now but please unresolved if you think otherwise.